common/src/main/java/org/opentestsystem/rdw/admin/common/security/PermissionScope.java

```diff
@@ -18,6 +18,11 @@
     public static final PermissionScope STATEWIDE = new PermissionScope();
 
     /**
+     * Empty permission scope flyweight
+     */
+    public static final PermissionScope EMPTY = new PermissionScope(false);
```

Comment:
I ported the explicit statewide PermissionScope implementation from RDW_Reporting in order to have a clean no-arg constructor for the Permission class.